src/Database/Driver/Sqlserver.php

```diff
@@ -64,8 +64,14 @@ public function connect()
             return true;
         }
         $config = $this->_config;
+
+        // PDO::ATTR_PERSISTENT is not allowed by SQL Server PHP drivers.
+        // See: https://github.com/Microsoft/msphpsql/issues/65
+        if (isset($config['persistent']) && $config['persistent']) {
+            throw new \InvalidArgumentException('Sqlserver driver does not allow config setting: persistent');
```

Comment:
I adjusted the error message early today.  Is the new one O.K.?

Your newest proposal is similar to what I changed it to but your suggestion is not quite factually correct.  You may or may or may not have "persistent" connections through the SQL Server PDO drivers.  IMHO telling users they can't have it because of the PDO driver is misleading.  The functional equivalent of persistent connections is connection pooling which is controlled outside the PDO drivers.  I've added a link to some documentation below.

Or perhaps you would be happier with the following message which covers all of the bases?

_Enabling persistent connections in Sqlserver by setting the config key "persistent" to "true" is not supported, as connection persistence is not controlled through the SQL Server PDO PHP driver.   Instead the PDO PHP driver relies on ODBC connection pooling, which is configured at the ODBC driver level._